packages/wdio-cli/src/commands/run.js

```diff
@@ -0,0 +1,150 @@
+import fs from 'fs-extra'
+import path from 'path'
+
+import Launcher from './../launcher'
+import Watcher from './../watcher'
+import { missingConfigurationPrompt } from '../utils'
+
+/* instanbul ignore next */
+export const command = 'run <configPath>'
+
+/* instanbul ignore next */
+export const desc = 'Run your WDIO configuration file to initialize your tests.'
+
+export const builder = {
```

Comment:
can we move this into a constant file?

Reply:
The idea what to keep the `builder` object inside each command. This object is used by yargs to create options for the command.

If you still want I can move it to constants, should I move all the builder objects to the constant file as well?